lib/solarwinds_apm/config.rb

```diff
@@ -68,11 +74,37 @@ def self.config_from_env
     end
 
     def self.set_log_level
-      SolarWindsAPM::Config[:debug_level] = 3 unless (-1..6).cover?(SolarWindsAPM::Config[:debug_level])
+      log_level = (ENV['SW_APM_DEBUG_LEVEL'] || SolarWindsAPM::Config[:debug_level] || 3).to_i
+      SolarWindsAPM.logger.level = LOGGER_LEVEL_MAPPING[log_level] || 1 # default log level info
```

Comment:
Same question, can we use the Ruby Logger constant `INFO` insteadn of 1?

Reply:
changed